rootfs/usr/local/bin/ovpn_genconfig

```diff
@@ -0,0 +1,377 @@
+#!/bin/bash
```

Comment:
can we just add openvpn docker image releases (push to hub.docker) to software updates channel?